crates/rooch-framework/sources/coin.move

```diff
@@ -335,6 +338,25 @@ module rooch_framework::coin {
         amount: u256,
     ): Coin<CoinType> {
         let addr = signer::address_of(account);
+        withdraw_interal<CoinType>(ctx, addr, amount) 
+    }
+
+    #[private_generics(CoinType)]
```

Comment:
Also provide an extend function via private_generics for `initialize`?

Reply:
Indeed, `register` will be more clarify then `initialize`

Reply:
Ok, I will refactor in future PR.